eosio_build.sh

```diff
@@ -118,7 +118,7 @@
 		export SOFTFLOAT_ROOT=/usr/local/berkeley-softfloat-3
 		CXX_COMPILER=clang++
 		C_COMPILER=clang
-		BUILD_MONGODB=true
+		BUILD_MONGODB=false
```

Comment:
Can you make this an option to the eosio_build.sh. It would be nice if Jenkins could build it so that our mongodb dependent tests can run.